pymatgen/core/surface.py

```diff
@@ -165,131 +165,24 @@ def __init__(
-
-            grouped = [list(sites) for k, sites in itertools.groupby(to_move, key=equi_index)]
-
-            if len(to_move) == 0 or any(len(g) % 2 != 0 for g in grouped):
-                warnings.warn(
-                    "Odd number of sites to divide! Try changing "
-                    "the tolerance to ensure even division of "
-                    "sites or create supercells in a or b directions "
-                    "to allow for atoms to be moved!"
-                )
-                continue
-            combinations = []
-            for g in grouped:
-                combinations.append(list(itertools.combinations(g, int(len(g) / 2))))
-
-            for selection in itertools.product(*combinations):
-                species = [site.species for site in fixed]
-                frac_coords = [site.frac_coords for site in fixed]
-
-                for s in to_move:
-                    species.append(s.species)
-                    for group in selection:
-                        if s in group:
-                            frac_coords.append(s.frac_coords)
-                            break
-                    else:
-                        # Move unselected atom to the opposite surface.
-                        frac_coords.append(s.frac_coords + [0, 0, shift])  # noqa: RUF005
+    def __str__(self) -> str:
+        comp = self.composition
+        outs = [
+            f"Slab Summary ({comp.formula})",
+            f"Reduced Formula: {comp.reduced_formula}",
+            f"Miller index: {self.miller_index}",
+            f"Shift: {self.shift:.4f}, Scale Factor: {self.scale_factor}",
+            f"abc   : {' '.join(f'{i:0.6f}'.rjust(10) for i in self.lattice.abc)}",
+            f"angles: {' '.join(f'{i:0.6f}'.rjust(10) for i in self.lattice.angles)}",
+            f"Sites ({len(self)})",
+        ]
 
-                # sort by species to put all similar species together.
-                sp_fcoord = sorted(zip(species, frac_coords), key=lambda x: x[0])
-                species = [x[0] for x in sp_fcoord]
-                frac_coords = [x[1] for x in sp_fcoord]
-                slab = Slab(
-                    self.lattice,
-                    species,
-                    frac_coords,
-                    self.miller_index,
-                    self.oriented_unit_cell,
-                    self.shift,
-                    self.scale_factor,
-                    energy=self.energy,
-                    reorient_lattice=self.reorient_lattice,
-                )
-                slabs.append(slab)
-        s = StructureMatcher()
-        return [ss[0] for ss in s.group_structures(slabs)]
+        for idx, site in enumerate(self):
+            outs.append(f"{idx + 1} {site.species_string} {' '.join(f'{j:0.6f}'.rjust(12) for j in site.frac_coords)}")
+        return "\n".join(outs)
 
-    def is_symmetric(self, symprec: float = 0.1):
+    @property
+    def is_symmetric(self, symprec: float = 0.1) -> bool:
```

Comment:
Default value of `symprec` of `0.1` is different from the default value in `SpacegroupAnalyzer` (0.01)